packages/ts/react-crud/src/autogrid.tsx

```diff
@@ -23,16 +23,63 @@ import type PropertyStringFilter from './types/dev/hilla/crud/filter/PropertyStr
 import type Sort from './types/dev/hilla/mappedtypes/Sort';
 import Direction from './types/org/springframework/data/domain/Sort/Direction';
 
+type AllPropertyNames<TItem> = string & keyof TItem;
+
+type UnsupportedColumnDataTypes = unknown[];
```

Comment:
The property filtering doesn't quite match what is implemented in code. It suggests nested properties from `@ManyToOne` relations which are not included by this logic here: 
https://github.com/vaadin/hilla/blob/ecc473cfabc068803a9cada7b75760c4365c6b3d/packages/ts/react-crud/src/property-info.ts#L66-L91

The same would apply for any other object model not annotated with `@OneToOne`.

If we want to make this work, we would have to:
- Include nested properties from all object models
- But do not show them by default

Reply:
I was using `GridWithEntityReferences.tsx` for testing.